### ISSUE TITLE
[Question] Add manual error

### ISSUE THREAD

**Issue by remy-poirier** [dmtrKovalenko/material-ui-pickers#510]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers |    1.0.0-rc.10     |
| material-ui         |   @material-ui/core 1.2.1     |
| React               |     16.4.1    |
| Browser             |   67.0.3396.99   |
| Platform            |         |

## Question

Hi, I am using Redux-form in my app, and I would like to know if it's possible to integrate an `error` props to the DatePicker component, this way, redux and this component could communicate, and I will be able to get error message from redux directly in my component.
Because the default validation system does not fit well with Redux-form

**Comment by dmtrKovalenko** [comment]
Yes you can pass the same error for helperTest as for native material-ui TextField.
Note that this will override all validation performing from the pickers side so you can use `onError` callback to make errors be controllable
